src/components/AttentionBox/AttentionBox.tsx

```diff
@@ -118,7 +122,9 @@ const AttentionBox: React.FC<AttentionBoxProps> & {
             iconLabel={iconLabel}
           />
         )}
-        {text || children}
+        <Text size="small" paragraph>
+          {text || children}
+        </Text>
```

Comment:
I wonder if it should have been 
```tsx
text ? <Text size="small" paragraph>{text}</Text> : children
```

Reply:
if it is not a text, it will be a composition of text and other stuff and in that case we still prefer that the component receive our typography styles

Reply:
I agree, but it might affect it badly in case of ellipsis wrapping

Reply:
paragraph is without ellipsis by default... Attention box is not support ellipsis before and still not supporting it